msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -0,0 +1,462 @@
+---
+title: Create API based Message extension
+author: v-ypalikila
+description: Learn how to build an API message extension and API plugin using Teams developer portal and Teams Toolkit.
```

Comment:
Should probably remove mention of "API plugin" for now.

Reply:
removed API plugin.